src/TTMp32Gme/Build/FileHandler.pm

```diff
@@ -222,6 +222,12 @@ sub get_tiptoi_dir {
 				return $d;
 			}
 		}
+	} else {
+		my $mountPoint = "/mnt/tiptoi";
```

Comment:
This change is quite specific to your containerization solution. Other users may have the tiptoi mounted somewhere else (usually automounted by udisks in /media/$ENV{'USER'}/tiptoi). Could you please check that path as well? Also: could you please document this in README.md?